rpcs3/Emu/Memory/vm.cpp

```diff
@@ -1602,11 +1607,11 @@ namespace vm
 
 			g_locations =
 			{
-				std::make_shared<block_t>(0x00010000, 0x1FFF0000, 0x200), // main
+				std::make_shared<block_t>(0x00010000, 0x1FFF0000, 0x300), // main
```

Comment:
I hw tested and vm::main is really 64k pages, not 4k. mostly because it is the same as vm::user64 but thats a different story..